File: Julia Version/eveneigenvalueofprolate.jl
#Hamed Baghal Ghaffari
#2021-10-20
#This function calculates the eigenvalues of the even PSWFs
function eveneigenvalueofprolate(c,m,n)
    global W
    W=coefficientprolate(c,m,2*n-1);
    y=(W[1]*sqrt(2))/(linearcombevenlegendreatzero(c,m,n));
    return y
end

File: Python Version/VS Code py Format/eveneigenvalueofprolate.py
#Hamed Baghal Ghaffari
#2021-10-17
#This function calculates the eigenvalues of the even PSWFs.
from coefficientprolate import coefficientprolate
from linearcombevenlegendreatzero import linearcombevenlegendreatzero
import math
def eveneigenvalueofprolate(c,m,n):
    W=coefficientprolate(c,m,2*n)
    y=(W[0]*math.sqrt(2))/(linearcombevenlegendreatzero(c,m,n))
    return y


#print(eveneigenvalueofprolate(1,20,1))